bin/templates/cordova/lib/builders/ProjectBuilder.js

```diff
@@ -250,43 +249,24 @@ class ProjectBuilder {
         var wrapper = path.join(this.root, 'gradlew');
         var args = this.getArgs(opts.buildType === 'debug' ? 'debug' : 'release', opts);
 
-        return spawn(wrapper, args, { stdio: 'pipe' })
-            .progress(function (stdio) {
-                if (stdio.stderr) {
-                    /*
-                    * Workaround for the issue with Java printing some unwanted information to
-                    * stderr instead of stdout.
-                    * This function suppresses 'Picked up _JAVA_OPTIONS' message from being
-                    * printed to stderr. See https://issues.apache.org/jira/browse/CB-9971 for
-                    * explanation.
-                    */
-                    var suppressThisLine = /^Picked up _JAVA_OPTIONS: /i.test(stdio.stderr.toString());
-                    if (suppressThisLine) {
-                        return;
-                    }
-                    process.stderr.write(stdio.stderr);
-                } else {
-                    process.stdout.write(stdio.stdout);
-                }
-            }).catch(function (error) {
+        return execa(wrapper, args, { stdio: 'inherit' })
+            .catch(function (error) {
                 if (error.toString().indexOf('failed to find target with hash string') >= 0) {
                     return check_reqs.check_android_target(error).then(function () {
                         // If due to some odd reason - check_android_target succeeds
                         // we should still fail here.
-                        return Q.reject(error);
+                        throw error;
                     });
                 }
-                return Q.reject(error);
+                throw error;
```

Comment:
The `Q` changes I think should done in the dropping Q PR : https://github.com/apache/cordova-android/pull/833, but this type of work kinda blends together.

Reply:
The Promise here would be unwrapped anyway. To explicitly reject with a `Q` Promise is a bit misleading, as the result will always be an `execa` Promise. But we could leave this for #833 if you insist.